hack/generate-proto.sh

```diff
@@ -14,17 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cd $GOPATH/src/github.com/GoogleContainerTools/skaffold
-export CONTAINER_NAME=generate-proto
-docker build -t $CONTAINER_NAME -f hack/proto/Dockerfile --target generateFiles proto
-if [ $? -ne 0 ]; then
-   printf "\nError generating files.\n"
-   exit 1
-fi
-
 set -e
 
-docker run $CONTAINER_NAME cat pkg/skaffold/server/proto/skaffold.pb.go > proto/skaffold.pb.go
-docker run $CONTAINER_NAME cat pkg/skaffold/server/proto/skaffold.pb.gw.go > proto/skaffold.pb.gw.go
+export CONTAINER_NAME=generate-proto
+docker build -t $CONTAINER_NAME -f hack/proto/Dockerfile --target generateFiles proto
+docker run --rm $CONTAINER_NAME cat pkg/skaffold/server/proto/skaffold.pb.go > proto/skaffold.pb.go
```

Comment:
why are the proto files getting generated into `pkg/skaffold/server`? they've been moved out into their own `proto` package right?

Reply:
Let me TAL